test/operatortest.jl

```diff
@@ -10,11 +10,24 @@
 
 @assert coeffs(commutator(sigma_x, sigma_x)) == spzeros(2,2)
 
-######################################
-# Position & Momentum Operators Test #
-######################################
+####################################################
+# Position, Displacement & Momentum Operators Test #
+####################################################
 
 p = positionop(2)
 m = momentumop(2)
 @assert coeffs(commutator(sigma_x, p)) == spzeros(2,2)
 @assert coeffs(commutator(sigma_y, m)) == spzeros(2,2)
+
+csvd = coherentstatevec_inf(20,1)
+csvo = displaceop(20,1)*statevec(1,FiniteBasis(20))
+@test_approx_eq_eps coeffs(csvd)[1] coeffs(csvo)[1] 1e-8
+
+############################
+# Squeezing Operators Test #
+############################
+
+@assert squeezingop(2,1.0)== QuArray(eye(2))
+@assert squeezingop(lowerop(2), lowerop(2), 1.0) == squeezingop(2,1)
```

Comment:
This is redundant since the left hand side is the definition of `squeezingop`. You could call `@assert squeezingop(lowerop(2), QuArray(eye(2)), 2.0) == displacementop(2,1.0)`, no?